lib/streamlit/version.py

```diff
@@ -94,15 +94,15 @@ def should_show_new_version_notice():
     """
     if random.random() >= CHECK_PYPI_PROBABILITY:
         # We don't check PyPI every time this function is called.
-        LOGGER.debug("Skipping PyPI version check")
+        _LOGGER.debug("Skipping PyPI version check")
         return False
 
     try:
         installed_version = _get_installed_streamlit_version()
         latest_version = _get_latest_streamlit_version(timeout=1)
-    except BaseException as e:
+    except Exception as ex:
         # Log this as a debug. We don't care if the user sees it.
-        LOGGER.debug("Failed PyPI version check.\n%s", e)
+        _LOGGER.debug("Failed PyPI version check.\n%s", ex)
```

Comment:
```suggestion
        _LOGGER.debug("Failed PyPI version check.", , exc_info=ex)
```
I've never used exc_info, but I've seen it elsewhere, so for the sake of consistency, we should standardize it.